packages/worker/src/worker.ts

```diff
@@ -1027,6 +1052,12 @@ export class Worker<T extends WorkerSpec = DefaultWorkerSpec> {
             throw new TypeError('Got an activity task without a "variant" attribute');
           }
           parentSpan.setAttribute('taskToken', formattedTaskToken).setAttribute('variant', variant);
+          parentSpan.setAttribute('run_id', task.start?.workflowExecution?.runId ?? 'unknown');
```

Comment:
Let's be consistent in the attribute casing.

Reply:
These need to match what core uses as well. I hadn't changed task token yet but it should become snake case too